load-test/station.js

```diff
@@ -0,0 +1,169 @@
+	response = createBroker(rootUrl, params, tempId);
+	key = `initial step for station-broker ${getStationCode(id)} / ${getBrokerCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 201 || r.status === 404,
+	});
+
+	// Create creditor institution
+	response = createCreditorInstitution(rootUrl, params, tempId);
+	check(response, {
+		'createCreditorInstitution': (r) => r.status === 201 || r.status === 404,
+	});
+
+}
+
+function postcondition(params, id) {
+	const tempId = `STATION${id}`;
+
+	// remove creditor institution and broker used in the test
+	let response = deleteCreditorInstitution(rootUrl, params, tempId);
+	let key = `final step for station ${getStationCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+
+	response = deleteBroker(rootUrl, params, tempId);
+	key = `final step for station-broker ${getStationCode(id)} / ${getBrokerCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+}
+
+export default function (data) {
+	// 3. VU code (once per iteration, as many times as the test options require)
+	// VU code runs in the default() function, running for as long and as many times as the options define.
+	const token = vars.env === "local" ? "-" : getJWTToken(vars.tenantId, vars.clientId, vars.clientSecret, vars.resource);
+
+	const params = {
+		headers: {
+			'Content-Type': 'application/json',
+			'Authorization': `Bearer ${token}`
+		},
+	};
+
+	const tempId = `STATION${__VU}`;
+
+	precondition(params, __VU);
+
+	// Get stations
+	let response = getStations(rootUrl, params);
+	check(response, {
+		'getStations': (r) => r.status === 200,
+	});
+
+	sleep(0.5)
+
+	// Create station
+	response = createStation(rootUrl, params, __VU, getBrokerCode(tempId));
+	check(response, {
+		'createStation': (r) => r.status === 201,
+	});
+
+	// Get station
+	response = getStation(rootUrl, params, __VU);
+	check(response, {
+		'getStation': (r) => r.status === 200,
+	});
+
+	// Update station
+	response = updateStation(rootUrl, params, __VU, getBrokerCode(tempId));
+	check(response, {
+		'updateStation': (r) => r.status === 200,
+	});
+
+	// Create station relationship
+	response = createStationRelationship(rootUrl, params, tempId, __VU);
+	check(response, {
+		'createStationRelationship': (r) => r.status === 201 || r.status === 404,
+	});
+
+	sleep(0.5)
+
+	// Get creditor institutions
+	response = getStationCreditorInstitutions(rootUrl, params, __VU);
+	check(response, {
+		'getStationCreditorInstitutions': (r) => r.status === 200,
+	});
+
+	// Delete station relationship
+	response = deleteStationRelationship(rootUrl, params, tempId, __VU);
+	check(response, {
+		'deleteStationRelationship': (r) => r.status === 200,
+	});
+
+	// Delete station
+	response = deleteStation(rootUrl, params, __VU);
+	check(response, {
+		'deleteStation': (r) => r.status === 200,
+	});
+
+	postcondition(params, __VU);
+}
+
+export function teardown(data) {
```

Comment:
we evaluate if it's convenient to put a comment  [sharedarray](https://k6.io/docs/javascript-api/k6-data/sharedarray/) rather than declaring 2 empty functions for life-cycle and putting comment there.
To avoid unnecessary code

Reply:
yes, I preferred to leave empty structure as reminder for any future needs.